client/my-sites/themes/theme-preview.jsx

```diff
@@ -178,6 +178,10 @@ class ThemePreview extends Component {
 
 	renderPrimaryButton = () => {
 		const primaryOption = this.getPrimaryOption();
+		if ( ! primaryOption ) {
```

Comment:
I wonder if we could detect this if this file were TypeScript 🙄 (It might be better to get this file into `.ts` in the future)

Reply:
Agreed. Many parts of Calypso are still not using TypeScript. That said, as we work on these parts we are able to refactor and create new components in TypeScript.